tests/unit/specs/utils/bioschemasMapper.spec.js

```diff
@@ -379,13 +382,20 @@ describe('bioschemasMapper', () => {
       expect(actual).toStrictEqual(expected)
     })
   })
+  describe('mapCollectionsDataMissingDescription', () => {
+    it('should use name as description in case the collection doesn\t have one', () => {
```

Comment:
You could use " " for the sentence, than you can use the ' 
or write it out in full: does not.

Reply:
Changed